domains/liquid.json

```diff
@@ -1,9 +1,9 @@
 {
     "owner": {
         "username": "ThinLiquid",
-        "email": "ayomideilesanmi10@gmail.com"
+        "email": "thinliquid@proton.me"
     },
     "record": {
-        "CNAME": "glitch.edgeapp.net"
+        "CNAME": "9b0994e0-3427-49c7-8609-b7027badd871.id.repl.co"
```

Comment:
I can't reach this website.

Reply:
it takes a while to load cause its replit